frontend/public/components/monitoring/table-pagination.tsx

```diff
@@ -30,6 +44,21 @@ const TablePagination = ({
       perPage={perPage}
       perPageOptions={paginationOptions}
       variant={PaginationVariant.bottom}
+      toggleTemplate={LocalizedToggleTemplate}
+      titles={{
+        items: '',
+        page: '',
+        itemsPerPage: t('public~Items per page'),
+        perPageSuffix: t('public~per page'),
+        toFirstPage: t('public~Go to first page'),
+        toPreviousPage: t('public~Go to previous page'),
+        toLastPage: t('public~Go to last page'),
+        toNextPage: t('public~Go to next page'),
+        optionsToggle: t('public~Items per page'),
+        currPage: t('public~Current page'),
+        paginationTitle: t('public~Pagination'),
+        ofWord: t('public~of'),
```

Comment:
Can `ofWord` now be removed from here?

Reply:
Thanks for the explanation. It's unfortunate, but I'm not sure what a better solution would be, so probably keeping `ofWord` is reasonable.

Reply:
I discussed with @rebeccaalpert and I don't think there is a better solution with our current i18n approach, so I think this is OK to merge.